python/edgar_filing_searcher/parsers/main.py

```diff
@@ -15,11 +16,20 @@ def create_url_list(url_edgar_current_events):
     return parse_13f_filing_detail_urls(text_edgar_current_events)
 
 
-def send_data_to_db(company_row, edgar_filing_row, data_13f_table):
+def update_filing_counts(cik_no_list):
+    """This function counts the number of filings and adds it to the Company table"""
+    for cik_no in cik_no_list:
+        company_in_table = Company.query.filter_by(cik_no=cik_no).first()
+        filing_count = EdgarFiling.query.filter_by(cik_no=cik_no).count()
+        company_in_table.filing_count = filing_count
+        db.session.commit()
+
+
+def send_data_to_db(company_row, edgar_filing_row, data_13f_table_parameter):
```

Comment:
parameter? You should pretty much never include the variable type or designation in the name. Maybe data_13f_table_rows might be better.